plugins/module_utils/copy.py

```diff
@@ -64,6 +64,50 @@ def __init__(self, module, client):
         self.container_arg = {}
         if module.params.get("container"):
             self.container_arg["container"] = module.params.get("container")
+        self.check_mode = self.module.check_mode
+
+    def _run_from_pod(self, cmd):
+        try:
+            resp = stream(
+                self.api_instance.connect_get_namespaced_pod_exec,
+                self.name,
+                self.namespace,
+                command=cmd,
+                async_req=False,
+                stderr=True,
+                stdin=False,
+                stdout=True,
+                tty=False,
+                _preload_content=False,
+                **self.container_arg
+            )
+        except Exception as e:
```

Comment:
You can also hit an exception during the reading or later when you parse the YAML. I think it would be better to just move the try/except block outside of the function and wrap the `self._run_from_pod()` call.